modules/virt-creating-vm-import-cli.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms/virt-creating-vms-from-container-disks.adoc
+// * virt/virtual_machines/creating_vms/virt-creating-vms-from-web-images.adoc
+
+ifeval::["{context}" == "virt-creating-vms-from-web-images"]
+:url:
+:title-frag: from an image on a web page
+:a-object: an image
+:object: image
+:data-source: web page
+endif::[]
+ifeval::["{context}" == "virt-creating-vms-from-container-disks"]
+:container-disks:
+:title-frag: from a container disk
+:a-object: a container disk
+:object: container disk
+:data-source: container registry
+endif::[]
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-import-cli_{context}"]
+= Creating a VM {title-frag} by using the command line
+
+You can create a virtual machine (VM) {title-frag} by using the command line.
+
+When the virtual machine (VM) is created, the data volume with the {object} is imported into persistent storage.
+
+.Prerequisites
+
+* You must have access credentials for the {data-source} that contains the {object}.
+
+.Procedure
+
+. If the {data-source} requires authentication, create a `Secret` manifest, specifying the credentials, and save it as a `data-source-secret.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: data-source-secret
+  labels:
+    app: containerized-data-importer
+type: Opaque
+data:
+  accessKeyId: "" <1>
+  secretKey:   "" <2>
+----
+<1> Specify the Base64-encoded key ID or user name.
+<2> Specify the Base64-encoded secret key or password.
```

Comment:
The majority of instances in the docs have this as `base64-encoded`.

Reply:
ISG does not cover usage but [Wikipedia](https://en.wikipedia.org/wiki/Base64) capitalizes it. Definitely a gap in our style guides.